components/pkg-export-kubernetes/src/lib.rs

```diff
@@ -55,13 +55,37 @@ pub fn export_for_cli_matches(ui: &mut UI, matches: &clap::ArgMatches) -> Result
     let image = if !matches.is_present("NO_DOCKER_IMAGE") {
         export_docker::export_for_cli_matches(ui, &matches)?
     } else {
+        ui.status(
+            Status::Custom('☛', String::from("Skipping")),
+            "Docker image generation",
+        )?;
         None
     };
     let mut manifest = Manifest::new_from_cli_matches(ui, &matches, image)?;
 
     let mut write: Box<Write> = match matches.value_of("OUTPUT") {
-        Some(o) if o != "-" => Box::new(File::create(o)?),
-        _ => Box::new(io::stdout()),
+        Some(o) if o != "-" => {
+            ui.status(
+                Status::Creating,
+                format!("Kubernetes manifest file {}", o),
+            )?;
+            let file = Box::new(File::create(o)?);
+            ui.status(
+                Status::Created,
+                format!("Kubernetes manifest file {}", o),
+            )?;
+
+            file
+        }
+        _ => {
+            let stdout = Box::new(io::stdout());
+            ui.status(
```

Comment:
So, `ui.status` prints to stderr then?

Reply:
No but I'm curious what gave you that impression? `ui.status` isn't used to print the manifest to stdout.

Reply:
So if you pass `-` as an output and then in the shell you redirect the output to a file, you'll get both status reports and a manifest in the file?

Reply:
Yes, that is true but if you want only the manifest into a file, you should just use `-o` then.

Reply:
Then `-` is pointless, isn't it? I would say that outputting to stdout would be useful when you want to redirect an output in a shell to a file by appending (`>>`).

Reply:
Not really. Sometimes you just want to see what will be generated. It can also be used for logging the output for debugging where you want all the output in one place.